lib/clients/websocket.js

```diff
@@ -17,13 +17,11 @@ class WebsocketClient extends EventEmitter {
     { channels = null } = {}
   ) {
     super();
-    this.productIDs = Utils.determineProductIDs(productIDs);
+    this.productIDs = new Set(Utils.determineProductIDs(productIDs));
```

Comment:
Not sure what these sets are used for — why aren't we using plain objects?

Reply:
Sets are a better construct for unique lists (obviously), but if you don't like them, I'll clean them out.